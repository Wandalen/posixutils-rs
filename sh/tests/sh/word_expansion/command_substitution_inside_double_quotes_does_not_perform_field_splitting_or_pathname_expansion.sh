DIR=/*
FIELDS='1,2,3'
IFS=,
echo "$(echo $DIR)"
echo "$(echo $FIELDS)"