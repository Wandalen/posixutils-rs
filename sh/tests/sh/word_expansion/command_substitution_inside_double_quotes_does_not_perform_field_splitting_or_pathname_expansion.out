/bin /boot /dev /etc /home /lib /lib32 /lib64 /libx32 /lost+found /media /mnt /opt /proc /recovery /root /run /sbin /srv /sys /tmp /usr /var
1 2 3
